deltachat-ios/AppDelegate.swift

```diff
@@ -55,6 +55,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
         logger.info("➡️ didFinishLaunchingWithOptions")
 
+        // The NSE ("Notification Service Extension") must not run at the same time as the app.
+        // The other way round, the NSE is not started with the app running.
+        UserDefaults.setMainAppRunning()
+        var pollSeconds = 0
+        while UserDefaults.nseFetching && pollSeconds < 30 {
+            logger.info("➡️ wait for NSE to terminate")
```

Comment:
This could flood the log if it takes some seconds for the app to start. Is this intended?

Reply:
it think it is fine, and may even give additional hints.

should not happen too often anyways - flooding would be 30 lines max and would require (a) app being stated just the moment a notification coming in and (b) having lots to do in NSE